scripts/prune_lm_dir.py

```diff
@@ -21,6 +22,15 @@
                     'prune*X, with X <= 1.0, tells it to prune with X times the threshold '
                     'specified with the --threshold option.  EM specifies one iteration of '
                     'E-M on the model. ')
+parser.add_argument("--target-size", type=int, default=0,
+                    help="Target size of final LM after pruning. "
+                    "If setting this to a positive value, the --steps would be "
+                    "ignored and a few steps may be worked out util the size "
+                    "of pruned LM match the target-size.")
+parser.add_argument("--tolerance", type=float, default=0.05,
+                    help="Tolerance of actual size of final LM and the target-size. ")
+parser.add_argument("--max-attempt", type=int, default=20,
+                    help="Max attempts to find target size LM. ")
```

Comment:
Or, we could change the content of log from 'iter' to 'step', and use '--max-iter' option?